.github/pull_request_template.md

```diff
@@ -1 +1,4 @@
-Please add all notable changes to the "Unreleased" section of the CHANGELOG.
+Please add all notable changes to the "Unreleased" section of the CHANGELOG in the format:
+```
+(JIRA ticket) Summary of changes. [Issue or PR #](link to issue or PR)
```

Comment:
I think we usually put a dash in front of this too.

Reply:
ah yeah, good call

Reply:
updated